puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/release_packages.yaml

```diff
@@ -34,6 +34,7 @@
                   block: true
     publishers:
       - trigger-parameterized-builds:
-        - project: release_mash
+        # TODO This probably doesn't work
```

Comment:
You're probably right, but I am not sure how to make it better.

Reply:
well, actually, we set `current-parameters: true`, so `$major_version` is passed as a param already, so you'd just need to have a generic `foreman-release-pipeline(-trigger)` job here, that triggers the real `foreman-VERSION-release-pipeline` based on a param. when triggering via a Groovy pipeline, you can do fancy vars just fine :)

Reply:
I'm debating if it should trigger at all. Since you need to sign RPMs in between, there's a good chance you need to trigger it anyway. On the other hand, you can often sign RPMs while the (ARM) debs are building and it can continue.